components/HistoryMessages.tsx

```diff
@@ -6,25 +6,36 @@ export default function HistoryMessages({
   messages,
   user,
 }: {
-    messages: Message[];
-    user: User;
+  messages: Message[];
+  user: User;
 }) {
+  const formatTimestamp = (timestamp) => {
```

Comment:
I probably will add Typescript type declaration to the timestamp parameter indicating that it can accept either a string or a Date object (just for us to make the habit work with Typescript types) :  const formatTimestamp = (timestamp: string | Date) => {
Good Job @mahadaali3 ! 
@HamzaAli303